src/Telemetry/Telemetry/Telemetry.php

```diff
@@ -394,13 +394,21 @@ protected function get_option() {
 	 * @return array
 	 */
 	protected function get_send_data_args() {
+		$opt_in_user = get_option( Status::OPTION_NAME_USER_INFO, [] );
+
+		$args = [
+			'token'         => $this->get_token(),
+			'telemetry'     => wp_json_encode( $this->provider->get_data() ),
+			'stellar_slugs' => wp_json_encode( $this->opt_in_status->get_opted_in_plugins() ),
+		];
+
+		if ( ! empty( $opt_in_user ) ) {
+			$args['opt_in_user'] = wp_json_encode( $opt_in_user );
```

Comment:
Adding the additional key here would require adjustments on the server to receive it. I think just getting it into $args['telemetry'] would be what we need.

Reply:
Oooohhhh, ok. Yeah, plopping that into `telemetry` is better. I'll make that change and adjust the tests.